### PR TITLE
(FM-8190) convert module to litmus

### PR DESCRIPTION
**Pull request by DavidS**
At the moment this is a draft PR to appveyor-verify changes.

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -1,64 +1,15 @@
...
-
-.gitignore:
-  required:
-    - ---.project
-
-spec/default_facts.yml:
-  unmanaged: true
-
-# The registry module doesn't support parallel_spec
-appveyor.yml:
```
Comment (tphoney) [inline .sync.yml]: does appveyor need ignored ? will pdk update smash the file ?

File: Gemfile [hunk 2]
```diff
@@ -29,13 +29,7 @@ group :development do
...
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.3',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "github_changelog_generator",                              require: false, git: 'https://github.com/skywinder/github-changelog-generator', ref: '20ee04ba1234e9e83eb2ffb5056e23d641c7a018' if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.2.2')
-end
-group :system_tests do
-  gem "puppet-module-posix-system-r#{minor_version}", require: false, platforms: [:ruby]
-  gem "puppet-module-win-system-r#{minor_version}",   require: false, platforms: [:mswin, :mingw, :x64_mingw]
-  gem "beaker-testmode_switcher", '~> 0.4',           require: false
-  gem "master_manipulator",                           require: false
-  gem "puppet-blacksmith", '~> 3.4',                  require: false
+  gem 'pdk', git: 'git://github.com/puppetlabs/pdk.git'
```
Comment (tphoney) [inline Gemfile]: looks un-needed

File: appveyor.yml [hunk 3]
```diff
@@ -34,30 +34,6 @@ environment:
       PUPPET_GEM_VERSION: ~> 6.0
       RUBY_VERSION: 25-x64
       CHECK: parallel_spec
-    -
```
Comment (tphoney) [inline appveyor.yml]: have you tried the tests in appveyor ? the localhost tests for litmus

**Comment by tphoney** [comment]
missing the distelli file as well.

File: Gemfile (hunk 2 above)
Comment (DavidS) [reply Gemfile]: oh yeah, that was debugging from my side and needs to be removed before merge

File: .sync.yml (hunk 1 above)
Comment (DavidS) [reply .sync.yml]: I was expecting `pdk update` to do the right thing - I'll need to investigate appveyor/litmus integration more.

File: provision.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+default:
+  provisioner: vmpooler
+  images: ['win-2016-x86_64']
+release_checks:
```
[the remark this answers is not in the document]
Comment (DavidS) [reply provision.yaml]: I've added the `windows-server` entry; the others don't make sense as the module is windows-only.

Might want to revisit that if we ever add tests to verify that the module does not break unrelated stuff on other OS'

File: Gemfile (hunk 2 above)
Comment (DavidS) [reply Gemfile]: done.

File: appveyor.yml (hunk 3 above)
Comment (DavidS) [reply appveyor.yml]: I'm doing that now.

File: .sync.yml (hunk 1 above)
Comment (DavidS) [reply .sync.yml]: copied over the recommended files.

File: Gemfile (hunk 2 above)
Comment (DavidS) [reply Gemfile]: this was a workaround for PDK-1525

File: distelli-manifest.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+team-modules/puppetlabs-motd:
```
Comment (tphoney) [inline distelli-manifest.yml]: project name needs changed

Comment (DavidS) [reply distelli-manifest.yml]: @sheenaajay just mentioned that to me :grimacing:

**Comment by DavidS** [comment]
successful run against `release_checks` platforms:

2019-10-24_14MS+0100_1278x124

**Pull request merged**